packages/rocketchat-i18n/i18n/pt-BR.i18n.json

```diff
@@ -2168,6 +2170,7 @@
   "Message_AudioRecorderEnabled_Description": "Requer arquivos 'audio / mp3' para ser um tipo de mídia aceito dentro das configurações 'Upload de arquivo'.",
   "Message_BadWordsFilterList": "Adicione palavrões para a lista negra",
   "Message_BadWordsFilterListDescription": "Adicione palavrões separados por vírgula para filtrar",
+  "Message_Characther_Limit": "Limite do caráter da mensagem",
```

Comment:
FYI `Characther` -> `Character`, but its the same everywhere for this so it still functions fine

Reply:
Thanks, @dlassalle. I'll fix this typo.